train.sh

```diff
@@ -0,0 +1,121 @@
+#!/bin/bash
```

Comment:
Pushing for my own convenience during the training, but I'll remove before merging